app/authorization/server/functions/upsertPermissions.js

```diff
@@ -145,6 +145,9 @@ export const upsertPermissions = () => {
 		// allows to manage voip calls configuration
 		{ _id: 'manage-voip-call-settings', roles: ['livechat-manager', 'admin'] },
 		{ _id: 'manage-voip-contact-center-settings', roles: ['livechat-manager', 'admin'] },
+		// allows agent-extension association.
+		{ _id: 'managage-agent-extension-association', roles: ['admin'] },
+		{ _id: 'query-agent-extension-association', roles: ['livechat-manager', 'admin', 'livechat-agent'] },
```

Comment:
```suggestion
		{ _id: 'view-agent-extension-association', roles: ['livechat-manager', 'admin', 'livechat-agent'] },
```
Suggesting this since we usually use "view" verb as a prefix for settings with only read access

Reply:
Fixed.